src/config/menu.json

```diff
@@ -52,7 +52,7 @@
     },
     {
       "name": "Partnership",
-      "url": "/contact"
+      "url": "/blog"
```

Comment:
Partnership will go to /blog? 🤔

Reply:
It's just for reference I'll change it later